gems/sorbet-runtime/test/types/props/_props.rb

```diff
@@ -408,7 +398,98 @@ class OverrideProps3 < OverrideProps
         end
       end
 
-      assert(error.message.include?("Attempted to override a prop :b on class Opus::Types::Test::Props::PropsTest::OverrideProps3 that doesn't already exist"))
+      assert(error.message.include?("You marked the getter for prop :b as `override`, but the method `b` doesn't exist to be overridden."))
+    end
+
+    module ManualGetter
+      extend T::Sig
+
+      sig { overridable.returns(Integer) }
+      def a; 0; end
+    end
+
+    module ManualSetter
+      extend T::Sig
+      extend T::Helpers
+
+      sig { overridable.params(a: Integer).returns(Integer) }
+      def a=(a); 0; end
+    end
+
+    it 'allows overriding normal methods' do
+      class OverrideProps4 < T::Struct
+        include ManualGetter
+        include ManualSetter
+
+        prop :a, Integer, override: true
+      end
+    end
+
+    it 'allows overriding only one half' do
+      class OverrideProps5 < T::Struct
+        include ManualGetter
+
+        prop :a, Integer, override: :reader
+      end
+    end
+
+    it 'permits {allow_incompatible: true}' do
+      class OverrideProps6 < T::Struct
+        include ManualSetter
+
+        prop :a, Integer, override: {writer: {allow_incompatible: true}}
+      end
+    end
+
+    it 'accepts override: false' do
+      class OverrideNothing < T::Struct
+        prop :a, Integer, override: false
+      end
+    end
+
+    # This came up with props defined by Chalk::ODM::AbstractModel
```

Comment:
For posterity, better to say "Stripe's model definitions" because people will not know what Chalk::ODM is.